File: WPA.backend/WPA.backend/Entities/Planner.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace WPA.backend.Entities
{
    public class Planner
    {
        public int Id { get; set; }
        public string PartnerName { get; set; }
        public DateTime WeddingDate { get; set; }
        public string WeddingPlace { get; set; }
        public DateTime CeremonyDate { get; set; }
        public string CeremonyPlace { get; set; }
        public virtual IList<Guest> Guests { get; set; }
        public virtual IList<Expense> Expenses { get; set; }
        public virtual IList<Income> Incomes { get; set; }
    }
}

File: frontend/src/app/api/models/planner.ts
/* tslint:disable */
import { Expense } from './expense';
import { Guest } from './guest';
import { Income } from './income';
export interface Planner {
  ceremonyDate: string;
  ceremonyPlace?: string;
  expenses?: Array<Expense>;
  guests?: Array<Guest>;
  id: number;
  incomes?: Array<Income>;
  weddingDate: string;
  weddingPlace?: string;
}
